azurerm/internal/services/network/network_watcher_resource_test.go

```diff
@@ -16,81 +16,11 @@ import (
-			"withFilters":                testAccPacketCapture_withFilters,
-			"requiresImport":             testAccPacketCapture_requiresImport,
-		},
-		"ConnectionMonitor": {
-			"addressBasic":                   testAccNetworkConnectionMonitor_addressBasic,
-			"addressComplete":                testAccNetworkConnectionMonitor_addressComplete,
-			"addressUpdate":                  testAccNetworkConnectionMonitor_addressUpdate,
-			"vmBasic":                        testAccNetworkConnectionMonitor_vmBasic,
-			"vmComplete":                     testAccNetworkConnectionMonitor_vmComplete,
-			"vmUpdate":                       testAccNetworkConnectionMonitor_vmUpdate,
-			"destinationUpdate":              testAccNetworkConnectionMonitor_destinationUpdate,
-			"missingDestinationInvalid":      testAccNetworkConnectionMonitor_missingDestination,
-			"bothDestinationsInvalid":        testAccNetworkConnectionMonitor_conflictingDestinations,
-			"requiresImport":                 testAccNetworkConnectionMonitor_requiresImport,
-			"httpConfiguration":              testAccNetworkConnectionMonitor_httpConfiguration,
-			"icmpConfiguration":              testAccNetworkConnectionMonitor_icmpConfiguration,
-			"bothAddressAndVirtualMachineId": testAccNetworkConnectionMonitor_withAddressAndVirtualMachineId,
-			"endpointType":                   testAccNetworkConnectionMonitor_endpointDeprecated,
-			"updateEndpoint":                 testAccNetworkConnectionMonitor_updateEndpointIPAddressAndCoverageLevel,
-		},
-		"PacketCapture": {
-			"localDisk":                  testAccNetworkPacketCapture_localDisk,
-			"storageAccount":             testAccNetworkPacketCapture_storageAccount,
-			"storageAccountAndLocalDisk": testAccNetworkPacketCapture_storageAccountAndLocalDisk,
-			"withFilters":                testAccNetworkPacketCapture_withFilters,
-			"requiresImport":             testAccNetworkPacketCapture_requiresImport,
-		},
-		"FlowLog": {
-			"basic":                testAccNetworkWatcherFlowLog_basic,
-			"disabled":             testAccNetworkWatcherFlowLog_disabled,
-			"reenabled":            testAccNetworkWatcherFlowLog_reenabled,
-			"retentionPolicy":      testAccNetworkWatcherFlowLog_retentionPolicy,
-			"updateStorageAccount": testAccNetworkWatcherFlowLog_updateStorageAccount,
-			"trafficAnalytics":     testAccNetworkWatcherFlowLog_trafficAnalytics,
-			"version":              testAccNetworkWatcherFlowLog_version,
-		},
-	}
-
-	for group, m := range testCases {
-		m := m
-		t.Run(group, func(t *testing.T) {
-			for name, tc := range m {
-				tc := tc
-				t.Run(name, func(t *testing.T) {
-					tc(t)
-				})
-			}
-		})
-	}
-}
-
-func testAccNetworkWatcher_basic(t *testing.T) {
+func TestAccNetworkWatcher_basic(t *testing.T) {
 	data := acceptance.BuildTestData(t, "azurerm_network_watcher", "test")
 	r := NetworkWatcherResource{}
 
-	data.ResourceTest(t, r, []acceptance.TestStep{
+	data.ResourceSequentialTest(t, r, []acceptance.TestStep{
```

Comment:
is this working correctly? all the tests are failing in teamcity now, worse then before